test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.cs

```diff
@@ -55,6 +55,19 @@ public class HubConnectionTests
         }
 
         [Fact]
+        public async Task SendAsyncThrowsIfSerializingMessageFails()
```

Comment:
Is this the case that would have swallowed the exception before?

Reply:
Yes this was one of the cases. `SendAsync` wouldn't throw at all. Connection could be closed or not started and it wouldn't throw either.